packages/cactus-plugin-keychain-aws-sm/package-lock.json

```diff
@@ -0,0 +1,1065 @@
+{
+	"name": "@hyperledger/cactus-plugin-keychain-aws-sm",
+	"version": "0.5.0",
```

Comment:
@jagpreetsinghsasan Please update to `0.6.0` (just issued the release) and then we are ready to go.